Explorer/Assets/Scripts/ECS/Unity/Systems/ParentingTransformSystem.cs

```diff
@@ -0,0 +1,82 @@
+using Arch.Core;
+using Arch.Core.Extensions;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using CRDT;
+using CrdtEcsBridge.Components.Transform;
+using ECS.Abstract;
+using ECS.Unity.Systems;
+using System.Collections.Generic;
+using Unity.ECS.Components;
+using UnityEngine;
+
+[UpdateInGroup(typeof(SimulationSystemGroup))]
+[UpdateAfter(typeof(InstantiateTransformSystem))]
+[UpdateBefore(typeof(UpdateTransformSystem))]
+public partial class ParentingTransformSystem : BaseUnityLoopSystem
+{
+    private readonly QueryDescription queryDescription = new QueryDescription().WithAll<SDKTransform, Transform>();
+    private Transform sceneRootTransform;
+
+    private OrphanTransform orphanTransform;
+    private ParentTransform parentTransform;
+
+    public ParentingTransformSystem(World world, Dictionary<CRDTEntity, Entity> entitiesMap, Transform sceneRootTransform) : base(world)
```

Comment:
`Dictionary<CRDTEntity, Entity> entitiesMap` must be propagated as `IReadOnlyDictionary` as you don't intend to modify it